src/main/resources/default/templates/biz/model/sql.html.pasta

```diff
@@ -111,6 +134,14 @@
                     return false;
                 }
             });
+
+            $('#exportBtn').click(function() {
+                if ($('#exportQuery').val() === "") {
+                    addError("Cannot export empty query");
+                } else {
+                    $('#exportForm').submit();
+                }
+            });
```

Comment:
i'd rather on/off this listener together with the show/hide statement in the execute function to keep the logic together, but this is fine too